instrumentation/opentelemetry-instrumentation-aiopg/tests/test_aiopg_integration.py

```diff
@@ -322,7 +320,7 @@ def test_span_failed(self):
         self.assertIs(
             span.status.status_code, trace_api.status.StatusCode.ERROR
         )
-        self.assertEqual(span.status.description, "Test Exception")
+        self.assertEqual(span.status.description, "Exception: Test Exception")
```

Comment:
Instrumentation was duplicating start_as_current_span's exception handling and not formatting the description properly. After removing the duplicate code, had to update the assertion to expect description formatted by the context manager. Similar changes below.